src/command.rs

```diff
@@ -143,24 +171,28 @@ pub fn run() -> Result<()> {
             cmd,
             |(c, _, iq, tm), _| Ok((cmd.run(c, iq), tm)),
             |(c, _, iq, tm), _| Ok((cmd.run(c, iq), tm)),
+            |(c, _, iq, tm), _| Ok((cmd.run(c, iq), tm)),
```

Comment:
I know this might sound naughty but could those three closures be simplified with a macro? Same for the closures below.

Reply:
Too naughty I think, not worth it, but nice thought... xD